Velaptor/NativeInterop/OpenGL/IGLInvoker.cs

```diff
@@ -591,4 +581,15 @@ internal interface IGLInvoker : IDisposable
     /// <param name="pixels">[length: COMPSIZE(format,type,width,height)] Specifies a pointer to the image data in memory.</param>
     void TexImage2D<T>(GLTextureTarget target, int level, GLInternalFormat internalformat, uint width, uint height, int border, GLPixelFormat format, GLPixelType type, byte[] pixels)
```

Comment:
If you would not mind, can you please add the suppress message attribute shown below?  I figured since we are in here anyway, we might as well add it.

**Little Background:**
The message comes from csharpsquid which is a C# analysis tool that is used under the hood by [SonarLint](https://www.sonarsource.com/products/sonarlint/).  This tool is used for code analysis in the cloud for all pull requests as well as a plugin to the JetBrains Rider IDE which is what I use.  

I don't think you see these because you use VSCode, so this will only show if you use the plugin in Visual Studio or Rider.

```suggestion
    [SuppressMessage("csharpsquid", "S107", Justification = "Maintaining param count to match original API.")]
    void TexImage2D<T>(GLTextureTarget target, int level, GLInternalFormat internalformat, uint width, uint height, int border, GLPixelFormat format, GLPixelType type, byte[] pixels)
```